Description: Upstream changes introduced in version 1.0.1c-4~gost
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 openssl (1.0.1c-4~gost) squeeze-backports; urgency=low
 .
   * Rules for build GOST engine
   * Commented init GOST in OpenSSL config file
 .
 The person named in the Author field signed this changelog entry.
Author: Mezentsev Vadim <mezentsevvo@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- openssl-1.0.1c.orig/apps/openssl.cnf
+++ openssl-1.0.1c/apps/openssl.cnf
@@ -1,4 +1,4 @@
-openssl_conf = openssl_def
+#openssl_conf = openssl_def
 #
 # OpenSSL example configuration file.
 # This is mostly being used for generation of certificate requests.
@@ -349,11 +349,12 @@ tsa_name		= yes	# Must the TSA name be i
 				# (optional, default: no)
 ess_cert_id_chain	= no	# Must the ESS cert id chain be included?
 				# (optional, default: no)
-[openssl_def]
-engines = engine_section
-[engine_section]
-gost = gost_section
-[gost_section]
-engine_id = gost
-default_algorithms = ALL
-CRYPT_PARAMS = id-Gost28147-89-CryptoPro-A-ParamSet
+#[openssl_def]
+#engines = engine_section
+#[engine_section]
+#gost = gost_section
+#[gost_section]
+#engine_id = gost
+#dynamic_path = /usr/lib/ssl/engines/libgost.so
+#default_algorithms = ALL
+
